templates/github/.ci/ansible/Containerfile.j2.copy

```diff
@@ -7,6 +7,12 @@ ADD {{ item.source }} {{ item.source }}
 {% endif %}
 {% endfor %}
 
+{%- for item in plugins -%}
+{%- if item.name == "pulp_container" -%}
+RUN dnf install -y podman
```

Comment:
why not having it by default?

Reply:
Good idea. I opened a PR for this. https://github.com/pulp/pulp-oci-images/pull/203